pdr_backend/cli/arg_feed.py

```diff
@@ -159,12 +184,24 @@ def _unpack_feeds_str(feeds_str: str) -> List[ArgFeed]:
 
     pairs = ArgPairs.from_str(pairs_list_str)
 
-    feeds = [
-        ArgFeed(exchange_str, signal_str, pair_str, timeframe_str)
-        for signal_str in signal_str_list
-        for pair_str in pairs
-        for timeframe_str in timeframe_str_list
-    ]
+    if isinstance(threshold_str_list[0], ArgThreshold):
+        if len(pairs) != len(threshold_str_list):
```

Comment:
I would suggest moving this check in a different function, only called if the threshold exists. It would help readability.

Reply:
not only the check, but maybe we could even return argfeeds_with_thresholds() or argfeed_without_thresholds() and clean up even more